api-model/src/main/java/io/enmasse/address/model/KubeUtil.java

```diff
@@ -73,6 +73,11 @@ public static String sanitizeWithUuid(String name, String uuid) {
         return sanitizeWithUuid(name, uuid, addressPattern);
     }
 
+    public static String infraUuid(String namespace, String name) {
+        int hash = (namespace + "." + name).hashCode();
```

Comment:
Isn't String#hashCode collision a concern?  Would a digest be a better choice?

Reply:
@k-wall I'm not sure I understand what you mean by digest, isn't that just the output of the hash function? I can update it to create a md5 hash if thats what you mean. In any case, we can't really use too many bytes to represent the UUID, because it will clash with length restrictions on service names which has a 64 byte length restriction.

Originally I thought an 8 byte hash would be sufficiently unique.

Reply:
By digest, I did mean an MD5 or SHA of the `namespace + "." + name` rather than relying on String#hashCode() (the algorithm can produce collisions with similar input).   As space is concern an abbreviated SHA (like used by git tools) would be sufficient for our needs, I think.

https://security.stackexchange.com/questions/52877/how-secure-is-javas-hashcode also points out that the implementation of String#hashCode may change across JVM releases or vendors.  We'd be safer with a known stable algorithm such as MD5 or SHA